management-server/src/main/kotlin/ut/isep/management/service/converter/applicant/ApplicantUpdateConverter.kt

```diff
@@ -0,0 +1,18 @@
+package ut.isep.management.service.converter.applicant
+
+import dto.applicant.ApplicantUpdateDTO
+import org.springframework.stereotype.Component
+import ut.isep.management.model.entity.Applicant
+import ut.isep.management.service.converter.UpdateConverter
+
+@Component
+class ApplicantUpdateConverter : UpdateConverter<Applicant, ApplicantUpdateDTO> {
+    override fun updateEntity(entity: Applicant, updateDTO: ApplicantUpdateDTO): Applicant {
+        return entity.apply {
+            updateDTO.name?.let { this.name = it }
+            updateDTO.status?.let { this.status = it }
+            updateDTO.preferredLanguage?.let { this.preferredLanguage = it }
+            updateDTO.score?.let { this.score = it }
```

Comment:
Can you update the score via the applicant Api endpoint? Wouldn't this not be better that system determines this by it self and when you approve a certain answer when you check the assessment results that it automatically updates the score. So then you shouldn't be able to update the score via the applicant Api directly

Reply:
Addressed in https://github.com/Alakind/ISEP/commit/d22ad8043225e091f27c7ecd001049b745356c04